Azure/sonic-buildimage
# sFlow : Upgrade hsflowd to v2.0.26-3

**- What I did**

Upgrade hsflowd to v2.0.26-3 to pick up recent fixes.

**- How I did it**

Update rules and makefile for sflow's hsflowd

**- How to verify it**

1. Check with https://sflow-rt.com/index.php that 64 bit counters are good (https://github.com/sflow/host-sflow/commit/c2030b0cd1eab0ac9c58d6b202c91d6ee15ac596)
2. Check that the last collector gets removed successfully (https://github.com/sflow/host-sflow/commit/dbf6cce334d171425f9a694a5feff55ff4131392). Fixes https://github.com/Azure/sonic-buildimage/issues/3989.

**- Description for the changelog**


sFlow : Upgrade hsflowd to v2.0.26-3

**- A picture of a cute animal (not mandatory but encouraged)**
